src/solver/simulation/sim_alloc_probleme_hebdo.cpp

```diff
@@ -182,9 +182,13 @@ void SIM_AllocationProblemePasDeTemps(PROBLEME_HEBDO& problem,
           .assign(linkCount, 0);
         variablesMapping.NumeroDeVariableCoutExtremiteVersOrigineDeLInterconnexion
           .assign(linkCount, 0);
-
-        variablesMapping.NumeroDeVariableDuPalierThermique
+        variablesMapping.NumeroDeVariableDuPalierThermique.assign(
+            study.runtime->thermalPlantTotalCount, 0);
+        variablesMapping.powerRampingIncreaseIndex
+          .assign(study.runtime->thermalPlantTotalCount, 0);
+        variablesMapping.powerRampingDecreaseIndex
           .assign(study.runtime->thermalPlantTotalCount, 0);
```

Comment:
By default, the variable doesn't exist

```suggestion
        variablesMapping.powerRampingIncreaseIndex
          .assign(study.runtime->thermalPlantTotalCount, -1);
        variablesMapping.powerRampingDecreaseIndex
          .assign(study.runtime->thermalPlantTotalCount, -1);
```